Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -1043,6 +1046,24 @@ def get_full_report_command(
         ]
     return command_results, is_command_finished
 
+def validate_sandbox_report(report_resources: list[dict[str, Any]]) -> str:
+    """This function checks for any error messages in the sandbox report.
```

Comment:
since we are using report and this is not generic, lets add a raw report for example.